core/src/test/java/org/elasticsearch/search/aggregations/metrics/InternalExtendedStatsTests.java

```diff
@@ -27,16 +27,16 @@
 import org.elasticsearch.search.aggregations.metrics.stats.extended.ParsedExtendedStats;
 import org.elasticsearch.search.aggregations.pipeline.PipelineAggregator;
 import org.elasticsearch.test.InternalAggregationTestCase;
-import org.junit.Before;
 
 import java.util.List;
 import java.util.Map;
 
 public class InternalExtendedStatsTests extends InternalAggregationTestCase<InternalExtendedStats> {
-    protected double sigma;
 
-    @Before
-    public void randomSigma() {
+    private double sigma;
+
+    @Override
+    public void setUp() throws Exception {
```

Comment:
AggregationsTests tests it using the default double value 0.0 for `sigma` which produces a valid XContent output that can be parsed back.